tests/vmi_multus_test.go

```diff
@@ -877,29 +877,116 @@ var _ = Describe("SRIOV", func() {
 	})
 })
 
+var _ = Describe("Macvtap", func() {
+	virtClient, err := kubecli.GetKubevirtClient()
+	tests.PanicOnError(err)
```

Comment:
Please refer to https://github.com/kubevirt/kubevirt/pull/3786 which atthends to not execute code in ginkgo containers body.

Reply:
I don't see the comment was addressed.

Reply:
Well, either I didn't understand you, or it is (I've moved that code into a `BeforeAll` block): https://github.com/kubevirt/kubevirt/pull/2935/commits/0391eda5202098806fafbb96a7acf32ed4cc468c

I'll squash this into the first commit that introduces that test (I agree this is located in the wrong commit).